### PR TITLE
docs: remove compatibility references; add deployment guide

### PR DESCRIPTION
**Pull request by MrMartinR**
Updated Min version required, 
Updated examples to year 2022 (just to make the docs more updated...)
Added 2 Deploying options, Bitnami (incomplete) and Ubuntu
Removed 2 files that probably are not used anymore
renamed a few files to match the kebab-case naming

(I am making a second PR, because for some reason some & characters disappeared.. )
Is my first time contributing, please check and if something wrong let me know/edit it.

### CODE REVIEW

File: _includes/graphql/objects.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ mutation createAGameScore {
       "clientMutationId": "anUniqueId",
       "gameScore": {
         "id": "R2FtZVNjb3JlOjZtdGlNcmtXNnY=",
-        "updatedAt": "2019-12-02T10:14:28.786Z",
-        "createdAt": "2019-12-02T10:14:28.786Z",
+        "updatedAt": "2022-12-02T10:14:28.786Z",
```
Comment (mtrezza) [inline _includes/graphql/objects.md]: This change is really just cosmetic and not really necessary, but I would not set it to a date in the future, that may cause confusion if someone implies anything into the date. I would change all dates to `2022-01-01T12:34:56.789Z` so in the future someone can replace them easily with a single batch replace, we could even automate that.

File: _includes/parse-server/database.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ If you have not used MongoDB before, we highly recommend familiarizing yourself
 
 The Mongo requirements for Parse Server are:
 
-* MongoDB version 3.6
-* An SSL connection is recommended (but not required).
+* MongoDB version 4.0 or newer
```
Comment (mtrezza) [inline _includes/parse-server/database.md]: We should remove any details here and refer to the README.md of the repo, because the compatibility is updated there frequently. Also the "SSL" recommendation is not a "requirement", so it should be removed. We have a security best practice page for that.

File: _includes/parse-server/database.md [hunk 3]
```diff
@@ -23,8 +23,15 @@ In order to allow for better scaling of your data layer, it is possible to direc
 
 The Postgres requirements for Parse Server are:
 
-* Postgres version 9.5
-* PostGIS extensions 2.3
+* PostgreSQL 11
```
Comment (mtrezza) [inline _includes/parse-server/database.md]: Same here as with mongodb, remove and compatibility details and refer to README.

File: _includes/parse-server/database.md [hunk 4]
```diff
@@ -44,8 +51,8 @@ Details about the configuration options can be found on [pg-promise](https://git
 * You will need to configure a [file adapter](#configuring-file-adapters) in order to store files.
 * Join tables are resolved in memory, there is no performance improvements using Postgres over MongoDB for relations or pointers.
 * Mutating the schema implies running ALTER TABLE, therefore we recommend you setup your schema when your tables are not full.
-* The postgres URL for 4.2.0 and below only supports the following configuration options:
+* The postgres URL for Parse 4.2.0 and below only supports the following configuration options:
```
Comment (mtrezza) [inline _includes/parse-server/database.md]: Remove Parse version specific reference, unless there is a specific reason.

File: _includes/parse-server/deploying-aws-bitnami.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+```bash
+cd stack/parse
+```
+```bash
+cat config.json
+```
+
+That will show Parse configuration details, save those details for later.
+
+<!-- WIP -->
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-bitnami.md]: I assume these comments will be removed before requesting review.

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+```bash
+cd /etc/postgresql/14/main/
+```
+We need to edit two files, `pg_hba.conf` and `postgresql.conf`
+```bash
+sudo nano pg_hba.conf
+```
+Scroll down the file and Add `host, all, all, 0.0.0.0/0, md5`, has to be the first line before `local, all, postgres, , peer`
+
+|  |  |  |  |  |
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: missing headlines

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+```bash
+sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
+```
+After that, we need to setup the configuration file, use your own `appId`, `masterKey` and `clientKey`, use random strings or some generator tool to create secured keys.
+```bash
+sudo nano -w config.json
+```
+
+```js
+// example
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: Remove "example" comment and describe what the code block is for in the doc text above.

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+```bash
+sudo npm install -g parse-dashboard
+```
+
+Once installed, you need to configure Parse Dashboard, go to `/usr/lib/node_modules/parse-dashboard/Parse-Dashboard/` and edit the file `parse-dashboard-config.json`
+```bash
+sudo nano -w parse-dashboard-config.json
+```
+```js
+// example
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: Same here, remove example comment and describe in doc text.

File: _includes/parse-server/development.md [hunk 9]
```diff
@@ -14,6 +14,8 @@ You can then link the parse-server module to the cloned repo and run `npm instal
 
 ```sh
 npm link parse-server path/to/cloned/repo
+```
+```sh
```
Comment (mtrezza) [inline _includes/parse-server/development.md]: combine code blocks into 1

File: _includes/parse-server/getting-started.md [hunk 10]
```diff
@@ -8,18 +8,25 @@ Parse Server is an open source backend that can be deployed to any infrastructur
 
 **Prerequisites**
 
-* Node 8 or newer
```
Comment (mtrezza) [inline _includes/parse-server/getting-started.md:11]: Remove any specific compatibility details and refer to README

File: _includes/parse-server/getting-started.md [hunk 11]
```diff
@@ -8,18 +8,25 @@ Parse Server is an open source backend that can be deployed to any infrastructur
...
-$ sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
-$ npm install -g mongodb-runner
-$ mongodb-runner start
-$ npm start
+sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
+```
+```bash
+npm install -g mongodb-runner
+```
+```bash
```
Comment (mtrezza) [inline _includes/parse-server/getting-started.md]: combine into 1 code block

File: _includes/parse-server/push-notifications-ios.md [hunk 12]
```diff
@@ -13,21 +13,21 @@ application.registerForRemoteNotifications()
 ```objc
 // Objective-C
 UIUserNotificationType userNotificationTypes = (UIUserNotificationTypeAlert |
-                                                UIUserNotificationTypeBadge |
```
Comment (mtrezza) [inline _includes/parse-server/push-notifications-ios.md:16]: I think that is/was the auto-format of Xcode for ObjC, if so I would revert these changes

File: _includes/parse-server/push-notifications.md [hunk 13]
```diff
@@ -44,7 +44,7 @@ You will need to obtain some credentials from FCM and APNS in order to send push
...
-If you are setting up push notifications on iOS, tvOS or macOS for the first time, we recommend you visit the [raywenderlich.com's Push Notifications tutorial](https://www.raywenderlich.com/123862/push-notifications-tutorial) or [appcoda.com's iOS Push tutorial](https://www.appcoda.com/push-notification-ios/) to help you obtain a production Apple Push Certificate. Parse Server supports the PFX (`.p12`) file exported from Keychain Access. Parse Server also supports the push certificate and key in `.pem` format. Token-based authentication instead of a certificate is supported as well.
+If you are setting up push notifications on iOS, tvOS or macOS for the first time, we recommend you visit the [raywenderlich.com's Push Notifications tutorial](https://www.raywenderlich.com/11395893-push-notifications-tutorial-getting-started) or [appcoda.com's iOS Push tutorial](https://www.appcoda.com/push-notification-ios/) to help you obtain a production Apple Push Certificate. Parse Server supports the PFX (`.p12`) file exported from Keychain Access. Parse Server also supports the push certificate and key in `.pem` format. Token-based authentication instead of a certificate is supported as well.
```
Comment (mtrezza) [inline _includes/parse-server/push-notifications.md:47]: We should probably write our own tutorial for this at some point.

File: _includes/parse-server/usage.md [hunk 14]
```diff
@@ -18,7 +18,7 @@ var api = new ParseServer({
 
 The parameters are as follows:
 
-* `databaseURI`: Connection string URI for your MongoDB.
+* `databaseURI`: Connection string URI for your MongoDB or PostgreSQL.
```
Comment (mtrezza) [inline _includes/parse-server/usage.md]: I would not mention a specific database, there are also other DB adapter for parse server out there.

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Comment by MrMartinR** [comment]
Hi Manuel, i will review all this tonight.

Btw.. I am doing this because Moumouls pushed me a bit 😂 on  my post in the forum, I barely know how to use git... so sorry for the mistakes... 
https://community.parseplatform.org/t/is-postgres-version-9-5-the-only-supported-version/2656/9

**Comment by mtrezza** [comment]
You are doing well, it's rare that a PR passes without any changes, so all good.

File: _includes/parse-server/database.md (hunk 4 above)
Comment (MrMartinR) [reply _includes/parse-server/database.md]: this case apply the specific reason isn't ? `only 4.2.0 and below supports....`

File: _includes/parse-server/deploying-aws-bitnami.md (hunk 5 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-bitnami.md]: Actually I haven't finish the Bitnami guide, because I was not sure if it will be published or not.. and I never did the Bitnami deployment myself, I just thought that could be useful for other users... not really sure if I have time/energy to test the Bitnami deployment and finish the guide to be honest... I will comment the link from the deploying.md file and I will decided later if I finish it...

File: _includes/parse-server/development.md (hunk 9 above)
Comment (MrMartinR) [reply _includes/parse-server/development.md]: Done. (This is a personal opinion: I did split the code, because feels weird, to see more than 1 line of code in the same block, usually the pages included the button to copy the code, and having multiple lines (or the $ prefix) makes the button useless)

File: _includes/graphql/objects.md (hunk 1 above)
Comment (MrMartinR) [reply _includes/graphql/objects.md]: I changed the dates to the recommended one "2022-01-01T12:34:56.789Z"

File: _includes/parse-server/database.md (hunk 2 above)
Comment (MrMartinR) [reply _includes/parse-server/database.md]: Done

File: _includes/parse-server/database.md (hunk 3 above)
Comment (MrMartinR) [reply _includes/parse-server/database.md]: Done

File: _includes/parse-server/getting-started.md (hunk 10 above)
Comment (MrMartinR) [reply _includes/parse-server/getting-started.md:11]: Done

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 6 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-ubuntu.md]: Done

File: _includes/parse-server/push-notifications-ios.md (hunk 12 above)
Comment (MrMartinR) [reply _includes/parse-server/push-notifications-ios.md:16]: I put it back the spaces

File: _includes/parse-server/push-notifications.md (hunk 13 above)
Comment (MrMartinR) [reply _includes/parse-server/push-notifications.md:47]: The link was redirected to the new url, I just updated the link.. I have no idea about the Push notifications, if some day I use/learn how to do it, maybe I will create the guide.. but for now _do not count on me_ for this... 🤣

File: _includes/parse-server/database.md (hunk 4 above)
Comment (mtrezza) [reply _includes/parse-server/database.md]: Ah yes, let's keep it then.

File: _includes/parse-server/deploying-aws-bitnami.md (hunk 5 above)
Comment (mtrezza) [reply _includes/parse-server/deploying-aws-bitnami.md]: We'd need to at least test the guide to ensure it's correct. What you could do it split the guide off i to a separate PR, so we can merge the other changes and at a later point merge the Bitnami guide once we know it works.

File: _includes/parse-server/development.md (hunk 9 above)
Comment (mtrezza) [reply _includes/parse-server/development.md]: That's a good point, not sure if we have such a copy/page button in our widget, but in any case I'd align with the test of the docs - if CLI commands are split into separate code blocks, then we can do that here too for consistency.

File: _includes/parse-server/deploying-aws-bitnami.md (hunk 5 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-bitnami.md]: I am a bit lost now...  I did remove the Bitnami guide file, so you can merge the PR now??

File: _includes/parse-server/development.md (hunk 9 above)
Comment (MrMartinR) [reply _includes/parse-server/development.md]: Yes, you have the copy button, I saw in some places the code is not split, combined looks nicer. (I have an internal fight about right-thing/functionality vs aesthetics.) 
I am not 100% sure right now, but I think if I copy the combined block and paste it in the terminal, executes the first line only or uses the rest of the lines as parameter or something weird happens.., and I remember having some drama in the past thinking that I run all the lines.

Also some blocks use **sh** others **bash** and others nothing **```**,  also inconsistency using **js** and **javascript**

(I am not trying to convince you or whatever... just my 2c)

File: _includes/parse-server/database.md [hunk 15]
```diff
@@ -8,25 +8,21 @@ The prefered database is MongoDB but Postgres is a great option if you're starti
 
 If you have not used MongoDB before, we highly recommend familiarizing yourself with it first before proceeding.
 
-The Mongo requirements for Parse Server are:
-
-* MongoDB version 3.6
-* An SSL connection is recommended (but not required).
+Check the [MongoDB requirements for Parse Server ](https://github.com/parse-community/parse-server#getting-started)
```
Comment (mtrezza) [inline _includes/parse-server/database.md]: Maybe let's compact this:

> If you have not used MongoDB before, we highly recommend familiarizing yourself with it first before proceeding. If this is your first time setting up a MongoDB instance, we recommend a Database-as-a-Service (DBaaS) like [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) or [ObjectRocket](https://objectrocket.com/) which provide fully managed MongoDB instances and can help you scale as needed.
> 
> Ensure that the MongoDB version is compatible with your version of Parse Server, for details see the [compatibility table](https://github.com/parse-community/parse-server#compatibility).

File: _includes/parse-server/database.md [hunk 16]
```diff
@@ -8,25 +8,21 @@ The prefered database is MongoDB but Postgres is a great option if you're starti
...
 
 When using MongoDB with your Parse app, you need to manage your indexes yourself. You will also need to size up your database as your data grows.
 
-In order to allow for better scaling of your data layer, it is possible to direct queries to a mongodb secondary for read operations.  See: [Mongo Read Preference](#using-mongodb-read-preference).
+In order to allow for better scaling of your data layer, it is possible to direct queries to a MongoDB secondary for read operations.  See: [MongoDB Read Preference](#using-mongodb-read-preference).
 
 ## Postgres
 
-The Postgres requirements for Parse Server are:
+Check the [Postgres requirements for Parse Server](https://github.com/parse-community/parse-server#getting-started)
```
Comment (mtrezza) [inline _includes/parse-server/database.md]: Same wording as above:

> Ensure that the Postgres version is compatible with your version of Parse Server, for details see the [compatibility table](https://github.com/parse-community/parse-server#compatibility).

File: _includes/parse-server/database.md [hunk 17]
```diff
@@ -44,8 +40,8 @@ Details about the configuration options can be found on [pg-promise](https://git
 * You will need to configure a [file adapter](#configuring-file-adapters) in order to store files.
 * Join tables are resolved in memory, there is no performance improvements using Postgres over MongoDB for relations or pointers.
 * Mutating the schema implies running ALTER TABLE, therefore we recommend you setup your schema when your tables are not full.
-* The postgres URL for 4.2.0 and below only supports the following configuration options:
+* The Postgres URL for Parse 4.2.0 and below only supports the following configuration options:
```
Comment (mtrezza) [inline _includes/parse-server/database.md]: More specific:

> The Postgres URL for Parse Server 4.2.0

because "Parse 4.2.0" could be understood as the Parse JS SDK.

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 18]
```diff
@@ -0,0 +1,192 @@
...
+
+1. Log into your AWS account or create a new one [AWS Account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+2. Go to AWS EC2 Dashboard
+3. Launch Instances
+4. On **Application and OS Images (Amazon Machine Image)** select `Ubuntu`
+5. On **Instance Type** select `t2.micro` (is ok for testing and small projects which is Free tier eligible)
+6. Create or select an existing `key pair`. (If you create a new one click `Download Key Pair`)
+7. Click `Launch Instance`
+8. On **EC2 Dashboard** select the new created instance and click `Security`
+9. On **Security Groups**, click on the security group _i.e.:sg-0fg55kj5588d63_
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: I don't think its necessary to add a security group example ID, maybe remove "_i.e.:sg-0fg55kj5588d63_"?

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 19]
```diff
@@ -0,0 +1,192 @@
...
+### Install Libraries and Dependencies
+Update the local package manager `apt`
+```bash
+sudo apt update
+```
+Install [NodeJS](https://nodejs.org)
+```bash
+sudo apt install nodejs
+```
+Check the install was ok, you should see the version installed. _i.e: v16.15.0_
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: I would not add a specific version here, this is something we'd have to maintain in the docs, because the version will look pretty old pretty soon. Maybe remove "`_i.e: v16.15.0_`"

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 20]
```diff
@@ -0,0 +1,192 @@
...
+Install Parse Dashboard globally
+```bash
+sudo npm install -g parse-dashboard
+```
+
+Once installed, you need to configure Parse Dashboard, go to `/usr/lib/node_modules/parse-dashboard/Parse-Dashboard/` and edit the file `parse-dashboard-config.json`
+```bash
+sudo nano -w parse-dashboard-config.json
+```
+This is an example of parse-dashboard.config.json, check the [full list of options](https://github.com/parse-community/parse-dashboard#configuring-parse-dashboard) for more details.
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: > Tis is an example of `parse-dashboard.config.json`

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 21]
```diff
@@ -0,0 +1,192 @@
...
+```js
+{
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md:162]: Is there 1 curly bracket too much or the indentation wrong?

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 22]
```diff
@@ -0,0 +1,192 @@
...
+sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
+```
+After that, we need to setup the configuration file, use your own `appId`, `masterKey` and `clientKey`, use random strings or some generator tool to create secured keys.
+```bash
+sudo nano -w config.json
+```
+This are the basic options of the config.json file, for the full list you can type `parse-server --help` or refer to the [Full Options Document](http://parseplatform.org/parse-server/api/master/ParseServerOptions.html) for more details.
+```js
+{
+  "appId": "luhYNQXysW6sS8S2s2c8PP9ZpLAf0gMb",
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: Not sure we should keep all these example values here because they may end up in actual projects; did you take them from somewhere else in the docs? I would just write:

>  "appId": "\<APP_ID\>",

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 23]
```diff
@@ -0,0 +1,192 @@
...
+sudo nano -w parse-dashboard-config.json
+```
+This is an example of parse-dashboard.config.json, check the [full list of options](https://github.com/parse-community/parse-dashboard#configuring-parse-dashboard) for more details.
+```js
+{
+{
+  "apps": [
+    {
+      "serverURL": "http://<your-server-IP-or-domain>:1337/parse",
+      "appId": "luhYNQXysW6sS8S2s2c8PP9ZpLAf0gMb",
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: Same here, I'd not use these example values

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 24]
```diff
@@ -0,0 +1,192 @@
...
+Start Parse Dashboard
+```bash
+parse-dashboard
+```
+or with the nohup command and specifying the configuration file, this option will keep the dashboard running even if you close the terminal
+```bash
+nohup parse-dashboard --dev --config parse-dashboard-config.json &
+```
+
+Check the Dashboard is running `http://host-ip-or-domain:4040`
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: > Ensure that Parse Dashboard is running by opening `http://<DOMAIN>:4040`.

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 25]
```diff
@@ -0,0 +1,192 @@
...
+
+```bash
+psql
+```
+
+```bash
+ALTER USER postgres password 'mypassword';
+```
+
+Quit psql typing `\q` ↩
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: Is this ` ↩` an actual sign that you put there, or is that just displayed by GitHub?
If we don't use that anywhere else, I would not put it here neither, for style consistency.

File: _includes/parse-server/getting-started.md [hunk 26]
```diff
@@ -8,18 +8,21 @@ Parse Server is an open source backend that can be deployed to any infrastructur
...
 
-* Node 8 or newer
-* MongoDB version 3.6
+* Node
+* MongoDB (to use Parse with MongoDB)
+* PostgreSQL (to use Parse with PostgreSQL)
 * Python 2.x (For Windows users, 2.7.1 is the required version)
 * For deployment, an infrastructure provider like Heroku or AWS
 
+Check the [requirements for Parse Server ](https://github.com/parse-community/parse-server#getting-started)
```
Comment (mtrezza) [inline _includes/parse-server/getting-started.md]: Same wording as above:

> Ensure that the Node.js version is compatible with your version of Parse Server, for details see the [compatibility table](https://github.com/parse-community/parse-server#compatibility).

File: _includes/parse-server/usage.md [hunk 27]
```diff
@@ -18,7 +18,7 @@ var api = new ParseServer({
 
 The parameters are as follows:
 
-* `databaseURI`: Connection string URI for your MongoDB.
+* `databaseURI`: Connection string URI for your database.
```
Comment (mtrezza) [inline _includes/parse-server/usage.md]: I don't think this is proper wording, "connection string" is just another word for "URI".

> * `databaseURI`: Connection string for your database.

File: package-lock.json [hunk 28]
```diff
@@ -1,8 +1,5446 @@
 {
   "name": "parsedocs",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (mtrezza) [inline package-lock.json]: Please revert clock file changes, unless you made some intentional changes?

**Review by mtrezza — CHANGES_REQUESTED** [review]
That's an amazing clean-up! Just found some last nits, then we can merge this.

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 21 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-ubuntu.md:162]: yep, there was an extra { 😅

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 22 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-ubuntu.md]: I was using random data because of the `This is an example...` but I think you are right, maybe not put the example because maybe people just copy-paste it, and later on, there will be millions of deployments with the same Ids/keys 🤣, I will use the placeholders.

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 24 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-ubuntu.md]: I will use <IP/DOMAIN> instead of <DOMAIN>, because actually the terminal always show the IPs instead of Domains, I think is good to know that people can use IPs or Domains in the configuration. and if I had to use only one, I will choose <IP>, but I will stick with both. (maybe <IP_or_DOMAIN> or <IP> or <DOMAIN>... 🤔

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 25 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-ubuntu.md]: I put it there, just for be sure to people press enter after typing, Maybe the 'right' thing should be: **`Quit psql typing \q and press enter/return`** mr obvious? not sure how dumb is the people (I am talking about me and the terminal-haters) ... 
Yeah.. I am gonna remove that thing...

File: _includes/parse-server/usage.md (hunk 27 above)
Comment (MrMartinR) [reply _includes/parse-server/usage.md]: I never heard about 'Connection string' in my life 🤣, I just changed MongoDB for database..  in this file, I am gonna  change `var` for `const` https://stackoverflow.com/questions/44212934/what-is-the-best-way-to-require-node-modules-var-or-const

File: package-lock.json (hunk 28 above)
Comment (MrMartinR) [reply package-lock.json]: I havent touch that file, I think was auto-generated when I run npm install or something... 🤔, I changed it to 1

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 22 above)
Comment (mtrezza) [reply _includes/parse-server/deploying-aws-ubuntu.md]: Yes, or if we use example values we make them very simple and obvious, like `exampleMasterKey`.

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 24 above)
Comment (mtrezza) [reply _includes/parse-server/deploying-aws-ubuntu.md]: Yes, `IP_OR_DOMAIN` looks readable.

**Comment by MrMartinR** [comment]
Note: on the file getting-started, I added DigitalOcean because they support the project, and why not, is a popular service anyway that probably many parse users use..
* For deployment, an infrastructure provider like Heroku, **DigitalOcean** or AWS

File: _includes/parse-server/getting-started.md [hunk 29]
```diff
@@ -8,18 +8,20 @@ Parse Server is an open source backend that can be deployed to any infrastructur
 
 **Prerequisites**
 
-* Node 8 or newer
-* MongoDB version 3.6
-* Python 2.x (For Windows users, 2.7.1 is the required version)
-* For deployment, an infrastructure provider like Heroku or AWS
+* Node
+* MongoDB (to use Parse with MongoDB)
```
Comment (mtrezza) [inline _includes/parse-server/getting-started.md]: > (to use Parse Server with ...

File: _includes/parse-server/getting-started.md [hunk 30]
```diff
@@ -8,18 +8,20 @@ Parse Server is an open source backend that can be deployed to any infrastructur
...
+* PostgreSQL (to use Parse with PostgreSQL)
+* For deployment, an infrastructure provider like Heroku, DigitalOcean or AWS
```
Comment (mtrezza) [inline _includes/parse-server/getting-started.md]: It seems that this guide actually refers to a local set-up for Parse Server, so maybe we should not mention a cloud services provider as a requirement. I would also mention them in order of popularity.

Let's add as a separate paragraph:

> The following guide describes how to set up Parse Server on your personal computer for local development. If you want to be able to access Parse Server from anywhere and make your app accessible publicly, you would want to deploy Parse Server to a cloud service provider like Amazon Web Services, Google Cloud, Microsoft Azure, Heroku or DigitalOcean. These providers vary in set-up complexity, configuration efforts, pricing model and required knowledge to secure your deployment. You can find guides for how to deploy Parse Server for specific providers in other parts of this documentation.
>
> ⚠️ Before making Parse Server accessible publicly, we strongly recommend to review all of your Parse Server configuration and read our [best practice](https://docs.parseplatform.org/parse-server/guide/#best-practice) guide. Failing to properly adapt your Parse Server configuration for a publicly accessible environment may make your deployment vulnerable to malicious intrusions, data leaks and unexpected cost increases.

Could you just change the best practice link to a relative link?

File: package-lock.json [hunk 31]
```diff
@@ -3,6 +3,5444 @@
   "version": "0.0.1",
   "lockfileVersion": 1,
   "requires": true,
+  "packages": {
```
Comment (mtrezza) [inline package-lock.json]: Could you completely revert any changes to this file? You can just copy/paste the original file there. It should not change when you run install to my knowledge, only if you remove it and then run install.

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Comment by mtrezza** [comment]
Many revisions in the reviews, but these docs were left untouched for so long that there was a lot of room for improvement. Thanks for your efforts to blow off the dust! I think this is an important refresh for new developers who rely on these guides.

File: package-lock.json (hunk 31 above)
Comment (MrMartinR) [reply package-lock.json]: Done!. I thought that only the  "lockfileVersion" was changed to 2, but I can see lots of changes...,  (no idea why that file was changed, I did 'nothing'... 🤷‍♂️)

File: _includes/parse-server/getting-started.md (hunk 30 above)
Comment (MrMartinR) [reply _includes/parse-server/getting-started.md]: I added the link to the "Deploying Parse Server", check if is ok.

File: _includes/parse-server/deploying-aws-ubuntu.md [hunk 32]
```diff
@@ -0,0 +1,187 @@
...
+sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
+```
+After that, we need to setup the configuration file, use your own `appId`, `masterKey` and `clientKey`, use random strings or some generator tool to create secured keys.
+```bash
+sudo nano -w config.json
+```
+This are the basic options of the config.json file, for the full list you can type `parse-server --help` or refer to the [Full Options Document](http://parseplatform.org/parse-server/api/master/ParseServerOptions.html) for more details.
+```js
+{
+  "appId": "<APP_ID>",
```
Comment (mtrezza) [inline _includes/parse-server/deploying-aws-ubuntu.md]: I think we could use the same values as below, `exampleMasterKey`, etc, so it's clear that these are the same values.

**Review by mtrezza — CHANGES_REQUESTED** [review]
Just one last change...

File: _includes/parse-server/deploying-aws-ubuntu.md (hunk 32 above)
Comment (MrMartinR) [reply _includes/parse-server/deploying-aws-ubuntu.md]: done, I also updated the Full Options Document link to point the last version of the file

File: package-lock.json [hunk 33]
```diff
@@ -70,6 +70,12 @@
             "minimist": "^1.2.5"
           }
         },
+        "minimist": {
```
Comment (mtrezza) [inline package-lock.json]: Still changes. Could you try to simply replace this file in your branch with the original file and commit?

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 33 above)
Comment (MrMartinR) [reply package-lock.json]: oh, sorry, I copied contents from 2 previous commits instead of last commit, should be ok now.. 😅🦄

Comment (mtrezza) [reply package-lock.json]: Yes, looks good now, thanks

**Review by mtrezza — APPROVED** [review]
Thanks for the clean-up! Very helpful to new developers to get started with Parse Platform!
